test/system/comment_test.rb

```diff
@@ -60,6 +60,15 @@ def setup
     click_on "Publish"
     find(".noty_body", text: "Comment Added!")
     find("p", text: "Awesome comment! :)")
+
+    # replying to the comment
+    first("p", text: "Reply to this comment...").click()
```

Comment:
Amazing!!!! Great work.

Reply:
However, I wonder if you could simulate clicking the "choose one" link and selecting an image? Is it possible? 

See how this test works! 

https://github.com/publiclab/plots2/blob/3faedc859a291cd3f907ceaa238ff6a7d268fa37/test/system/comment_test.rb#L81-L102

Reply:
> However, I wonder if you could simulate clicking the "choose one" link and selecting an image? Is it possible?
> 
> See how this test works!
> 
> https://github.com/publiclab/plots2/blob/3faedc859a291cd3f907ceaa238ff6a7d268fa37/test/system/comment_test.rb#L81-L102

Ok, willl do this.